specs/src/specs/params.md

```diff
@@ -31,7 +31,7 @@ are blocked by the `x/paramfilter` module.
 | distribution.WithdrawAddrEnabled | true | Enables delegators to withdraw funds to a different address. | True |
 | distribution.BaseProposerReward | 0 | Reward in the mint demonination for proposing a block. | True |
 | distribution.BonusProposerReward | 0 | Extra reward in the mint denomination for proposers based on the voting power included in the commit. | True |
-| gov.DepositParams.MinDeposit | 1000000000utia (1000 TIA) | Minimum deposit for a proposal to enter voting period. | True |
+| gov.DepositParams.MinDeposit | 100000000000utia (10000 TIA) | Minimum deposit for a proposal to enter voting period. | True |
```

Comment:
[nit] comma and underscores for readability 

```suggestion
| gov.DepositParams.MinDeposit | 100_000_000_000 utia (10,000 TIA) | Minimum deposit for a proposal to enter voting period. | True |
```

also the value for utia looks right per https://www.google.com/search?q=100000000000+%2F+10e6